integration/windows_hostprocess_test.go

```diff
@@ -124,3 +124,47 @@ func runHostProcess(t *testing.T, expectErr bool, image string, action hpcAction
 
 	action(t, cn, containerConfig)
 }
+
+func startAndTestContainer(t *testing.T, sb string, sbConfig *runtime.PodSandboxConfig, cnConfig *runtime.ContainerConfig) {
+	t.Log("Create the container")
+	cn, err := runtimeService.CreateContainer(sb, cnConfig, sbConfig)
+	require.NoError(t, err)
+
+	t.Log("Start the container")
+	_, err = t, runtimeService.StartContainer(cn)
+
+	if err != nil {
+		t.Fatalf("Starting container: %v failed", err)
+	}
+
+	t.Log("Stop the container")
+	require.NoError(t, runtimeService.StopContainer(cn, 0))
+
+	t.Log("Remove the container")
+	require.NoError(t, runtimeService.RemoveContainer(cn))
+}
+
+func TestArgsEscapedImagesOnWindows(t *testing.T) {
```

Comment:
Is there any reason we're testing with a hostprocess container instead of regular? Not that I think my command line parsing code isn't rock solid 🤣 but feel it'd be more useful to test against HCS were most of the bug reports had been for this.